enzy_htp/_interface/amber_interface.py

```diff
@@ -3286,6 +3304,51 @@ def make_mmpbgbsa_nc_file(
 
         self.convert_traj_to_nc(stru_esm.coordinate_list, temp_nc)
 
+    def make_mc_file(self, maa_region: StructureRegion, out_path: str):
+        """make the mc file for parameterization"""
+        maa: ModifiedResidue = maa_region.involved_residues[0]
+        
+        if not maa.is_connected():
+            raise AttributeError(f"maa is not connected; use init_connectivity() first.") 
+
+        # main chain
+        mc_atoms = maa.find_mainchain()
+        lines = ["HEAD_NAME: N", "TAIL_NAME: C"]
+        
+        for aa in mc_atoms:
+            lines.append(f"MAIN_CHAIN: {aa.name}")
+
+        # omit cap atom names
+        for cap in maa_region.caps:
+            for aa in cap.atoms:
+                lines.append(f"OMIT_NAME: {aa.name}")
+
+        # find pre_head and post_tail atom types
+        for cap in maa_region.caps:
+            if cap.link_atom.element == "N":
```

Comment:
We should determine the name of the link atom instead of element here. Imagine there are more than just C-ter and N-ter cap but also caps on other nitrogens.